.github/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+**Description:**
+<!--- What does this PR do? -->
+
+**Motivation and context:**
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+<!--- If this change depends on another PR, please link to that PR here. -->
+
+**How has this been tested?**
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Reviewers will test your PR in at least this way. -->
+
+**How long should this take to review?**
+<!--- Please estimate if this is a small, average, or large PR. -->
+<!--- Take into account both the size and complexity of the changes. -->
+<!--- Also note if this is a timely PR that should be reviewed by a certain date. -->
```

Comment:
Do we expect people to give an estimate in some amount of time or just say something like "small" or "large"? I think time estimates are really difficult to give (especially for other people). In the latter case it might be useful to have checkboxes where you just click the right one. It would also be helpful to have some indication of what counts as small, average, and large. E.g. small (less than 10 lines changed and 3 files touched), average (neither small or large), large (more that 150 lines changed)